lmms_eval/models/chat/openai_compatible.py

```
@@ -39,89 +30,117 @@ class OpenAICompatible(OpenAICompatibleSimple):
-
-            res.append(response_text)
-            pbar.update(1)
 
-            if self.continual_mode is True:  # Cache the response
+        for batch_requests in batched_requests:
+            batch_payloads = []
+            batch_doc_uuids = []
+            batch_responses = []
+
+            for req in batch_requests:
+                ctx, doc_to_messages, gen_kwargs, doc_id, task, split = req.args
```

Ruff (0.13.1)

Unpacked variable `ctx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)